app/assets/javascripts/components/subjects_taught.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var toggleSubjectValues = function(event) {
```

Comment:
This will pollute the global namespace. Better to either wrap this in an immediately invoked function or define it inside the `DOMContentLoaded` callback (my preference). Also, it's more readable to use the `function name(args)` form, rather than variable assignment, IMO.